weasel/about.py

```diff
@@ -1,3 +1,2 @@
-__version__ = "3.5.0"
 __projects__ = "https://github.com/explosion/projects"
 __projects_branch__ = "v3"
```

Comment:
This will be in spacy instead, let's just remove this whole file?

Reply:
It's used for the `clone` command... We should probably remove it nonetheless, but let's do that in a separate PR?